src/frontend/src/handler/mod.rs

```diff
@@ -52,6 +52,7 @@ pub mod create_mv;
 pub mod create_schema;
 pub mod create_sink;
 pub mod create_source;
+pub mod create_sql_function;
```

Comment:
Why not put in the same `create_function` file?

Reply:
explained in  https://github.com/risingwavelabs/risingwave/pull/14139#discussion_r1436174036